### ISSUE TITLE
Bug in emoji code?

### ISSUE THREAD

**Issue by arohl** [swiftbar/SwiftBar#335]
My plugin that alert me to upcoming cricket games yields

```
:australia: v :england: 3:03 | symbolize=false
---
Australia vs England starts on Sun Oct 09 at 4:10 PM | color = royalblue
```

The Australian flag renders correctly but the English one renders as the text `:england:` not the flag. I note that if I unquote the text in for the flag in this report I get the rendered flag - :england: on my Mac and I thought both the GitHub website and SwiftBar use the same rendering engine!

**Comment by melonamin** [comment]
Hey @arohl,

For whatever reason I have implemented it this way:
```
    ":gb:": "🇬🇧",
    ":uk:": "🇬🇧",
```

So if you replace `:england:` to `:uk:` you'll see the flag.

I've added `:england:` as an option as well, it will be available in the next release

**Comment by melonamin** [comment]
Note to self - don't work on GitHub issues before morning coffee 😅

Got it, I've added a proper `:england:` 🏴󠁧󠁢󠁥󠁮󠁧󠁿 flag, will be available with the 1.4.4 release, which I hope to publish soon.

**Comment by arohl** [comment]
😃
